qml/main.qml

```diff
@@ -40,6 +40,143 @@ ApplicationWindow {
 
     cover: Qt.resolvedUrl("pages/CoverPage.qml")
 
+    ListModel {
+        id: regions
+
+        ListElement {
+            name: QT_TR_NOOP("Finland")
+            apiName: "finland"
+        }
+        ListElement {
+            name: QT_TR_NOOP("Helsinki and the capital region (HRT)")
+            identifier: "helsinki"
+            apiName: "hsl"
+            boundarycirclelat: 60.169
+            boundarycirclelon: 24.940
+        }
+        ListElement {
+            name: QT_TR_NOOP("Tampere region (Nysse)")
+            identifier: "tampere"
+            apiName: "finland"
+            boundarycirclelat: 61.498
+            boundarycirclelon: 23.759
+        }
+        ListElement {
+            name: QT_TR_NOOP("Turku region (Föli)")
+            identifier: "turku"
+            apiName: "finland"
+            boundarycirclelat: 60.451
+            boundarycirclelon: 22.267
+        }
+        ListElement {
+            name: QT_TR_NOOP("Hämeenlinna region")
+            identifier: "hameenlinna"
+            apiName: "waltti"
+            boundarycirclelat: 60.997
+            boundarycirclelon: 24.465
+        }
+        ListElement {
+            name: QT_TR_NOOP("Iisalmi")
+            identifier: "iisalmi"
```

Comment:
Iisalmi has two lines and very few buses during weekdays. By short testing, routing results show confusing extra lines 5171, 5211 and 20. These might be some long-distance buses since [opas.matka.fi](https://opas.matka.fi/) gives similar results. I think a route planner for this small amount of buses is a bit overkill too. Therefore I would leave Iisalmi out entirely.

Reply:
It's true that Iisalmi has very little public transport. I think it would still be useful as an option, as the application makes it easier to check the timetables and it doesn't really hurt to have it as an option. The route planning, of course, isn't particularly useful there.